src/main/java/org/takes/facets/auth/PsToken.java

```diff
@@ -117,14 +120,19 @@ private PsToken(final SiHmac sign, final long seconds) {
         String head = "";
         while (headers.hasNext()) {
             final String nexthead = headers.next();
-            if (nexthead.trim().startsWith("Bearer")) {
+            final UncheckedText trimmed = new UncheckedText(
+                new Trimmed(new TextOf(nexthead))
+            );
+            if (trimmed.asString().startsWith("Bearer")) {
                 head = nexthead;
                 break;
             }
         }
         final String dot = "\\.";
         if (!head.isEmpty()) {
-            final String jwt = head.split(" ", 2)[1].trim();
+            final String jwt = new UncheckedText(
```

Comment:
@fanifieiev Following the request above, you could use `Split` here too

Reply:
> @fanifieiev Following the request above, you could use `Split` here too

@paulodamaso `Split` does not support `limit` which is required here. I registered a ticket for that a time ago. https://github.com/yegor256/cactoos/issues/1251